### PR TITLE
Add launching import via admin

### PR DESCRIPTION
**Pull request by igor-kamil**
just to note:
- new **migration**

to **listen the queue**, the suggested approach is 
```
php artisan queue:listen --tries=2
```

### CODE REVIEW

**Comment by rastislav-chynoransky** [comment]
Please update `.env.example` to reflect expected `QUEUE_DRIVER` value

File: app/Jobs/ImportCsv.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $import = $this->import; //temporary fix because of PHP5.6 and T_PAAMAYIM_NEKUDOTAYIM error
+
+        $import->status = $import::STATUS_IN_PROGRESS;
```
Comment (rastislav-chynoransky) [inline app/Jobs/ImportCsv.php]: Use classname to access its constant.

**Comment by igor-kamil** [comment]
@rastislav-chynoransky thank you for code review. 
I fixed all the mentioned issues.
Except "view related methods to model". 
Well, I am also not fan of them.
Any better approach?

**Comment by rastislav-chynoransky** [comment]
@igor-kamil The simplest solution i can think of is creating a function that is passed to the view together with other data. If the mapping is used anywhere else, only then I would create a global helper.

**Comment by rastislav-chynoransky** [comment]
@igor-kamil Only one suggestion to views, do not use non-escaped output if not necessary

**Pull request merged**